packages/frontend/src/components/CollectionCard/CollectionCard.scss

```diff
@@ -1,14 +1,26 @@
 .collection-card {
     background-color: white;
-    padding: 12px 20px 12px 20px;
-    box-sizing: border-box;
-    display: table;
-    width: 100%;
-    flex-direction: column;
     border-radius: 1px;
     border: 1px solid #c5cede;
+    box-sizing: border-box;
     box-shadow: 1px 1px 2px rgba(90, 117, 152, 0.3);
+    display: table;
+    flex-direction: column;
+    position: relative;
+    padding: 12px 20px 12px 20px;
+    transition: all 200ms;
+    width: 100%;
+
+    &:hover {
+        box-shadow: 1px 3px 10px rgba(90, 117, 152, 0.4);
```

Comment:
I know our convention is to use 2 spaces, but this file already was using 4 spaces so I left it as is so as to reduce the number of line changes for this PR